tests/conftest.py

```diff
@@ -1,12 +1,12 @@
-try:
-    from unittest.mock import AsyncMock
-except ImportError:
-    # Python 3.7 and below don't include unittest.mock.AsyncMock. Hence,
-    # we need to resolve to a package on pypi.
-    from asynctest import CoroutineMock as AsyncMock
+import sys
+
+from unittest.mock import AsyncMock
 
 import pytest
 
+if sys.version_info < (3, 8):
+    from asynctest import CoroutineMock as AsyncMock
+    
```

Comment:
The changes in this PR aim to drop support for Python 3.7. Therefore, this code is not needed. 
Even if you run the code with Python 3.7, it would fails. Since the dependency `asynctest` has been removed.

Reply:
Don't forget to remove `import sys`, since that module isn't used anymore.